cli/common/socket.go

```diff
@@ -75,6 +75,18 @@ func ReadFromConn(conn net.Conn, endOfOutput string) ([]byte, error) {
 	return data, nil
 }
 
+func endOfOutputFound(data []byte, endOfOutput string) bool {
+	endOfOutputLen := len(endOfOutput)
+	dataLen := len(data)
+
+	if dataLen < endOfOutputLen {
+		return false
+	}
+
+	endOfData := data[dataLen-endOfOutputLen:]
+	return string(endOfData) == endOfOutput
```

Comment:
Maybe hasPostfix is better here?

Reply:
Of course we don't want to cast whole data to string (in case it leads to copy of whole data buffer). But this "endswith" implementation isn't clear enough IMO.